.travis.yml

```diff
@@ -21,9 +21,10 @@ install:
 
   - conda env create -n test-environment -f environment.yml
   - source activate test-environment
-  - python setup.py install
+  - pip install -e .
```

Comment:
No need to install the package twice. Just the second form is sufficient.

Read the relevant section at https://setuptools.readthedocs.io/en/latest/setuptools.html
```
As you can see, the extras_require argument takes a dictionary mapping names of “extra” features, to strings or lists of strings describing those features’ requirements. These requirements will not be automatically installed unless another package depends on them (directly or indirectly) by including the desired “extras” in square brackets after the associated project name.
```

Reply:
Removed in the new commit.